### PR TITLE
fix(test): Remove failing compatibility tests

### PR DESCRIPTION
**Pull request by connorgorman**
## Description

3.74 and 4.1 are both out of support and are both affected by the issue where network entities are too large so remove them from the version compatibility matrix.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

Version compatibility test

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: scripts/ci/jobs/gke_version_compatibility_tests.py [hunk 1]
```diff
@@ -65,16 +64,7 @@
...
-# Support exception for latest central and sensor 3.74 as per
-# https://issues.redhat.com/browse/ROX-18223
-support_exceptions = [
-    ChartVersions(
-        central_version=latest_tag,
-        sensor_version=get_latest_helm_chart_version_for_specific_release(
-            "stackrox-secured-cluster-services", Release(major=3, minor=74)
-        ),
-    )
-]
```
Comment (vikin91) [inline scripts/ci/jobs/gke_version_compatibility_tests.py:77]: Could we at least have it commented out? Would be cool to have these tests once we fix the issue in 3.74

**Review by vikin91 — APPROVED** [review]
This is okay, but let's maybe keep that 3.74 code commented out until we release a new patch for 3.74, so that we can reenable it later.

File: scripts/ci/jobs/gke_version_compatibility_tests.py (hunk 1 above)
Comment (connorgorman) [reply scripts/ci/jobs/gke_version_compatibility_tests.py]: Done

**Pull request merged**